### ISSUE TITLE
Update jscodeshift version

### ISSUE THREAD

**Issue by bpfoster** [pocka/vue-docgen-loader#17]
The version of `jscodeshift` being brought in via this library is old and brings in a high vulnerability:

```
braces  <3.0.3
Severity: high
Uncontrolled resource consumption in braces - https://github.com/advisories/GHSA-grv7-fg5c-xmjg
No fix available
node_modules/jscodeshift/node_modules/braces
  micromatch  <=4.0.7
  Depends on vulnerable versions of braces
  node_modules/jscodeshift/node_modules/micromatch
    jscodeshift  0.3.20 - 0.13.1
    Depends on vulnerable versions of micromatch
    node_modules/jscodeshift
      vue-docgen-loader  >=1.3.0-beta.0
      Depends on vulnerable versions of jscodeshift
      node_modules/vue-docgen-loader
```

`jscodeshift` is up to v17.0.0, but even v0.14.0 will resolve this.

**Comment by pocka** [comment]
Thank you for bringing this up, I'll check their changelog then update the dependency version.

**Comment by pocka** [comment]
Addressed in v2.0.1

**Comment by bpfoster** [comment]
Thank you!
